Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -284,15 +258,15 @@ public static Image GetGrayScaleVersion(this Image original)
             using (Graphics g = Graphics.FromImage(newBitmap))
             {
                 //create some image attributes
-                using (ImageAttributes attributes = new ImageAttributes())
-                {
-                    //set the color matrix attribute
-                    attributes.SetColorMatrix(ColorMatrix);
-
-                    //draw the original image on the new image
-                    //using the grayscale color matrix
-                    g.DrawImage(original, new Rectangle(0, 0, original.Width, original.Height),
-                        0, 0, original.Width, original.Height, GraphicsUnit.Pixel, attributes);
+                using (ImageAttributes attributes = new ImageAttributes())
+                {
+                    //set the color matrix attribute
+                    attributes.SetColorMatrix(ColorMatrix);
+
+                    //draw the original image on the new image
+                    //using the grayscale color matrix
+                    g.DrawImage(original, new Rectangle(0, 0, original.Width, original.Height),
+                        0, 0, original.Width, original.Height, GraphicsUnit.Pixel, attributes);
                 }
```

Comment:
Hm, I can't see any change in these lines... maybe some artifact of the diff tool.

Reply:
True, can't remember doing something here, maybe a quirck in the diff tool or a formatting thingy